jax/numpy/lax_numpy.py

```diff
@@ -2610,6 +2610,26 @@ def polyval(p, x):
   return y
 
 
+def trim_zeros(a):
+  first = 0
+  for i in a:
+    if i != 0:
+      break
+    first = first + 1
+  return a[first:]
+
+@_wraps(np.polymul)
```

Comment:
Use `lax_description` to describe what `trim_leading_zeros` does: essentially, setting it to True makes the output match that of numpy, but prevents the function from being able to be used in compiled code.